.github/workflows/gpt_code_review.yml

```diff
@@ -18,7 +18,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
           # Optional
-          OPENAI_API_ENDPOINT: https://api.openai.com/v1
           MODEL: gpt-4 # https://platform.openai.com/docs/models
-          max_tokens: 10000
-          MAX_PATCH_LENGTH: 10000 # if the patch/diff length is large than MAX_PATCH_LENGTH, will be ignored and won't review. By default, with no MAX_PATCH_LENGTH set, there is also no limit for the patch/diff length.
```

Comment:
I was getting an error:     "message": "This model's maximum context length is 8192 tokens. However, you requested 10220 tokens (220 in the messages, 10000 in the completion). Please reduce the length of the messages or completion.",

I removed this to avoid the explicit length request. Can I add back in `MAX_PATCH_LENGTH` without triggering this issue, as long as I continue to omit `max_tokens`?